lib/bugsnag/integrations/delayed_job.rb

```diff
@@ -54,43 +56,35 @@ def error(job, error)
                 }
                 p[:object][:id] = object.id if object.respond_to?(:id)
               end
-              add_active_job_details(p, payload)
+              if payload.respond_to?(:job_data) && payload.job_data.respond_to?(:[])
+                [:job_class, :arguments, :queue_name, :job_id].each do |key|
+                  if (value = payload.job_data[key.to_s])
+                    p[key] = value
+                  end
+                end
+              end
               overrides[:job][:payload] = p
             end
 
-            ::Bugsnag.notify(error, true) do |report|
+            ::Bugsnag.notify(exception, true) do |report|
               report.severity = "error"
               report.severity_reason = {
                 :type => ::Bugsnag::Report::UNHANDLED_EXCEPTION_MIDDLEWARE,
-                :attributes => FRAMEWORK_ATTRIBUTES
+                :attributes => {
+                  :framework => "DelayedJob"
+                }
```

Comment:
Is it possible to set the app_type here? Like you do with rake: https://github.com/bugsnag/bugsnag-ruby/blob/56f387ef36c157a2b6fbf7d9e6807d21d1e62393/lib/bugsnag/integrations/rake.rb#L14